src/tiny-slider.js

```diff
@@ -545,7 +545,13 @@ export var tns = function(options) {
     rect = div.getBoundingClientRect();
     width = rect.right - rect.left;
     div.remove();
-    return width || getClientWidth(el.parentNode);
+    if (width) {
+        return width;
+    } else if (el.parentNode.parentNode !== null) {
```

Comment:
it may happen that the parentNode is not null but the document element and then the error is triggered too